cartridges/openshift-origin-cartridge-haproxy/usr/bin/update-cluster

```diff
@@ -111,8 +111,11 @@ fi
 local_ep=$local_ip:$local_port
 
 # Add the local gear
+# For cookie consistency with remote gears we'll use the gear's DNS name here,
+# whic takes the form:   gearname-domain
```

Comment:
Thanks, will fix the `which` and re-push.

Remote gears's cookies are set in line 76: `$gear_name` there comes from the arguments in the `update-cluster` invocation, which are a list of `gear_dns_name|endpoint`; line 49 sets `gear_name` based on the DNS name of the gears, so this change does the same with the local gear.